elasticapm/traces.py

```diff
@@ -385,13 +385,14 @@ def ensure_parent_id(self) -> str:
         return self.trace_parent.span_id
 
     def to_dict(self) -> dict:
-        self.context["tags"] = self.labels
+        context = self.context.copy()
```

Comment:
`copy()` is shallow, which is fine as long as we only operate on the top level dictionary (which, if I didn't overlook something, we currently do). Maybe still worth a note here to change to `deepcopy()` if we ever have to modify data deeper in the context object.